site/src/components/checkbox.md

```diff
@@ -58,14 +58,40 @@ Here is an example of a group of checkboxes. Default spacing is added between ea
 </label>
 ```
 
-### Size
+### Sizes
 
-`.checkbox-sm` is used with `.checkbox` to display a smaller checkbox.
+`.checkbox-sm`/`.checkbox-lg` used in conjunction with `.checkbox` makes checkbox a little bigger or smaller than the default.
 
 ```html
 <label class="checkbox checkbox-sm">
-	<input type="checkbox" checked name="checkbox-small" />
+	<input type="checkbox" name="checkbox-size" />
 	<span class="checkbox-check" role="presentation" aria-hidden="true"></span>
-	<span class="checkbox-text">Checked</span>
+	<span class="checkbox-text">Checkbox small</span>
+</label>
+<label class="checkbox">
+	<input type="checkbox" name="checkbox-size" />
+	<span class="checkbox-check" role="presentation" aria-hidden="true"></span>
+	<span class="checkbox-text">Checkbox default</span>
+</label>
+<label class="checkbox checkbox-lg">
+	<input type="checkbox" name="checkbox-size" />
+	<span class="checkbox-check" role="presentation" aria-hidden="true"></span>
+	<span class="checkbox-text">Checkbox large</span>
+</label>
+```
+
+### Validation states
+
+`is-invalid` helps with visual reflection of invalid validation state.
+
```

Comment:
We can use `.is-invalid` to reflect an invalid validation state. Note that the state should also be display to the user in another way, since by itself color is not enough.

Reply:
Updated